src/AutoRest.CSharp/Common/Output/Models/Types/ModelTypeProvider.cs

```diff
@@ -53,7 +53,7 @@ internal sealed class ModelTypeProvider : SerializableObjectType
         public override ObjectTypeProperty? AdditionalPropertiesProperty => throw new NotImplementedException();
 
         public ModelTypeProvider(InputModelType inputModel, string defaultNamespace, SourceInputModel? sourceInputModel, TypeFactory? typeFactory = null, InputModelType[]? derivedTypes = null, ObjectType? defaultDerivedType = null)
-            : base(inputModel.Namespace ?? defaultNamespace, sourceInputModel)
+            : base(GetDefaultNamespace(defaultNamespace), sourceInputModel)
```

Comment:
Think about this scenario: model-namespace is false, and user defines namespace in cadl. Is there any issue?

Should it be: 
`GetDefaultNamespace(inputModel.Namespace ?? defaultNamespace)` ?

Reply:
No. now we will ignore model's namespace defined in cadl, we will use common namespace which is `<serviceNamespace>/<serviceNamespace>.Models`